.circleci/config.yml

```diff
@@ -2,13 +2,20 @@ version: 2
 jobs:
 
   # If you want to deploy your container to Docker Hub, you must
+  # define environment variables in the CircleCI web interface for your project
   #  1. define the CONTAINER_NAME environment variable for the project
+  #     Specifically, this is the name of the Docker Hub <ORG>/<NAME> to push
   #     If not defined, will use the repository that is being built from.
-  #  2. Define the DOCKER_TAG if you have preference. If not defined, will use
+  #     to in the case that you have provided Docker credentials. 
```

Comment:
did a line get lost here in the reformatting?

Reply:
Nope! I just re-ordered the variables based on what I thought was more important (toward the top, so the user reads it first). The entire section is now:

```bash
  # If you want to deploy your container to Docker Hub, you must
  # define environment variables in the CircleCI web interface for your project
  #  1. define the CONTAINER_NAME environment variable for the project
  #     Specifically, this is the name of the Docker Hub <ORG>/<NAME> to push
  #     If not defined, will use the repository that is being built from.
  #     to in the case that you have provided Docker credentials. 
  #  2. Define DOCKER_USER and DOCKER_PASS (for push to Docker Hub, if wanted)
  #     If you don't define these in CircleCI settings it won't be pushed.
  #  3. Optionally define the DOCKER_TAG. If not defined, will use
  #     the commit associated with the build.
  #  4. By default, the build assumes wanting to use the local notebook here.
  #     If you want to build from a **different** Github repository, then define
  #     the variable REPO_NAME as the name of the repository that you 
  #     want to build. That can be done in this file under build:environment,
  #     or again online in the CircleCI environment variable settings. 
```